src/main/resources/processes/MessageIT.bpmn20.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:activiti="http://activiti.org/bpmn" id="Definitions_098ju69" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Activiti Modeler" exporterVersion="3.0.0-beta">
+  <bpmn:process id="shouldDeliverMessagesViaRestApi" isExecutable="true">
+    <bpmn:sequenceFlow id="SequenceFlow_0zk2l68" sourceRef="StartEvent_1" targetRef="Task_05bhhuh" />
+    <bpmn:startEvent id="StartEvent_1" name="startMessage">
+      <bpmn:outgoing>SequenceFlow_0zk2l68</bpmn:outgoing>
+      <bpmn:messageEventDefinition messageRef="Message_1v73ptz" />
+    </bpmn:startEvent>
+    <bpmn:userTask id="Task_05bhhuh" name="Task">
+      <bpmn:incoming>SequenceFlow_0zk2l68</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_1m22yb2</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:sequenceFlow id="SequenceFlow_0kppnex" sourceRef="BoundaryEvent_0vax021" targetRef="ExclusiveGateway_0yhejky" />
+    <bpmn:boundaryEvent id="BoundaryEvent_0vax021" name="boundaryMessage" attachedToRef="Task_05bhhuh">
+      <bpmn:outgoing>SequenceFlow_0kppnex</bpmn:outgoing>
+      <bpmn:messageEventDefinition messageRef="Message_1tdomgh" activiti:correlationKey="${execution.processInstanceBusinessKey}" />
+    </bpmn:boundaryEvent>
+    <bpmn:sequenceFlow id="SequenceFlow_1vsdq0e" sourceRef="IntermediateThrowEvent_1wte9f3" targetRef="EndEvent_1yxwnmm" />
+    <bpmn:intermediateCatchEvent id="IntermediateThrowEvent_1wte9f3" name="catchMessage">
+      <bpmn:incoming>SequenceFlow_0bc2ui7</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_1vsdq0e</bpmn:outgoing>
+      <bpmn:messageEventDefinition messageRef="Message_065r3wd" activiti:correlationKey="${execution.processInstanceBusinessKey}" />
+    </bpmn:intermediateCatchEvent>
+    <bpmn:endEvent id="EndEvent_1yxwnmm" name="endMessage">
+      <bpmn:incoming>SequenceFlow_1vsdq0e</bpmn:incoming>
+      <bpmn:messageEventDefinition messageRef="Message_1sg2fls" activiti:correlationKey="${execution.processInstanceBusinessKey}" />
```

Comment:
@igdianov, is it normal that the end event is referencing a message?

Reply:
@erdemedeiros  It is normal, it is throw message end event. See the diagram in the description.